docs/reference-guides/interactivity-api/core-concepts/undestanding-global-state-local-context-and-derived-state.md

```diff
@@ -789,14 +789,14 @@ store( 'myPlugin', {
 
 `getServerContext()` allows you to subscribe to changes in the **local context** that occur during client-side navigation. This function is analogous to `getServerState()`, but it works with the local context instead of the global state.
 
+The `getServerContext()` function returns a read-only reactive object. This means that any [callbacks](/docs/reference-guides/interactivity-api/api-reference.md#accessing-data-in-callbacks) you have defined that watch the returned object will only trigger when the value returned by the function changes. If the value remains the same, the callback will not re-trigger.
+
 Consider a quiz that has multiple questions. Each question is a separate page. When the user navigates to a new question, the server provides the new question and the time left to answer all the questions.
 
 ```php
-<?php
-wp_interactivity_context( 'myPlugin', array(
+<div <?php echo wp_interactivity_data_wp_context( 'myPlugin', array(
```

Comment:
`wp_interactivity_data_wp_context` has the namespace at the end, and it's optional.

Reply:
👍 . Done in 377d59cb4bef0901aecfd3b56eae55a77e3ddbc6